mvp.feature

```diff
@@ -21,13 +21,14 @@ Feature: MVP features for NVA
     And they see their name in the Menu
     And they see the Connect Author dialog
 
-  @wip
+  @1205
   Scenario: User connects Author
     Given that the user logs in with Feide for the first time
     When they click OK in the Connect Author dialog
```

Comment:
Trenger et step før man trykker OK for Connect Author. Enten at User allerede har koblet en Author eller at User velger en Author fra listen som blir presentert.

Reply:
Tenker dette bør være en egen PR